### PR TITLE
Create stubs for LA recs

### LINKED ISSUE #11: Create a new file for leey46's hometown
# LA
- [x] Create a branch
- [x] Add commits (make changes)
- [ ] Pull request
- [ ] Review/Collaborations
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by leey46**
closes #11

### CODE REVIEW

File: leey46.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Taipei
+
+### Places to eat
+
```
Comment (angelaRWDS) [inline leey46.md]: could you please recommend a dumpling place?

File: leey46.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Taipei
```
Comment (chuoc) [inline leey46.md]: Is Taipei in LA?

Comment (leey46) [reply leey46.md]: It's similar in many ways 😆

**Review by angelaRWDS — APPROVED** [review]

**Comment by leey46** [comment]
🎈 good job 👍

**Pull request merged**